tutorials/editor/upgrading_to_godot_4.rst

```diff
@@ -573,7 +577,7 @@ converter doesn't support updating existing setups:
 +---------------------+-----------------------+----------------------------------------------------------------------------+
 | ToolButton          | Button                | ToolButton was Button with the **Flat** property enabled by default.       |
 +---------------------+-----------------------+----------------------------------------------------------------------------+
-| YSort               | Node2D                | Node2D has a new **Y Sort** property in 4.0.                               |
+| YSort               | CanvasItem            | CanvasItem has a new **Y Sort Enabled** property in 4.0.                           |
```

Comment:
Should the second column be replaced with "Node2D, Control" since `CanvasItem` is an abstract class?

Reply:
cc @Calinou

Reply:
Indeed, I'd replace it with `Node2D or Control` to be more explicit. Also, the table columns should be adjusted to match (this is a reStructuredText requirement).

Reply:
Done.